### ISSUE TITLE
Is it possible to back up only a specific file?

### ISSUE THREAD

**Issue by sreehax** [libimobiledevice/libimobiledevice#831]
Is it possible to back up only a specific file from the iDevice, specifying the file either by filename or hash?

**Comment by mexmer** [comment]
no, ios backup doesn't have such function. you need to do complete backup, then pick the file you want.

there is no such parameter for creating backup.
ios backup works simple
you tell device to do backup, device send you files it wants to have backed up.
there is no option to specify what files you want.
